examples/protonect/CMakeLists.txt

```diff
@@ -166,6 +166,28 @@ IF(ENABLE_OPENCL)
     LIST(APPEND RESOURCES
       src/opencl_depth_packet_processor.cl
     )
+
+    INCLUDE(CheckCSourceRuns)
```

Comment:
move this part to a separate cmake file in cmake_modules similar to SetupLibfreenect2Threading.cmake?! keeps the main CMakeLists.txt cleaner?